Makefile

```diff
@@ -32,14 +31,15 @@ inplace:
 	$(PYTHON) setup.py build_ext -i
 
 test-code:
-	$(NOSETESTS) -s nilearn $(NOSETESTS_OPTIONS)
+	python -m $(TEST_RUNNER) --pyargs nilearn --cov=nilearn -s -vv --durations=0
+
 test-doc:
-	$(NOSETESTS) -s --with-doctest --doctest-tests --doctest-extension=rst \
-	--doctest-extension=inc --doctest-fixtures=_fixture `find doc/ -name '*.rst'`
+	$(TEST_RUNNER) -s --doctest-glob='*.rst' --doctest-modules
```

Comment:
I think this will result in all the other tests (besides doctest) being run twice, because they are already run in test-code and you removed `find doc/ -name '*.rst'`

I don't think pytest has an option to collect only doctests yet

https://github.com/pytest-dev/pytest/issues/4726

Reply:
WOuldn't `--doctest-glob` replace `find doc/ -name '*.rst'`?
I will try `--doctest-glo='doc/*.rst'` .

Reply:
if we follow what is done in scikit-learn and nilearn master, doctests in docstrings will be tested in `test-code` and those in rst files will be tested in test-doc

to do this:
- add the `--doctest-modules` option to pytest in test-code, either in the makefile or by adding a `[pytest]` section to the `setup.cfg` 
- select only .rst files as input to pytest in `test-doc` by invoking like `pytest [options] $(find -name "*.rst")`

Reply:
> WOuldn't `--doctest-glob` replace `find doc/ -name '*.rst'`?

no, one is telling it in which files to look for doctests while the other is the input (which when not provided defaults to the current directory ie all of nilearn in this case)

Reply:
Yeah didn't work when I tried it